assets/src/components/site-scan-results/plugins-with-amp-incompatibility.js

```diff
@@ -40,6 +49,26 @@ export function PluginsWithAmpIncompatibility( { className, slugs = [], ...props
 			className={ classnames( 'site-scan-results--plugins', className ) }
 			{ ...props }
 		>
+			{ showHelpText && (
+				<p>
+					{ createInterpolateElement(
+						__( 'Because of plugin issues we’ve uncovered, you may want to <a>review and suppress plugins</a>.', 'amp' ),
```

Comment:
Aside: With `createInterpolateElement`, what happens when you have more than one `<a>` in your string?

Reply:
You can use whatever tag name you wish, e.g. you could do something like this:

```js
createInterpolateElement( 'Visit <ArchiveLink>archive</ArchiveLink> or return to <HomeLink>homepage</HomeLink>', {
	ArchiveLink: <a href="/archive" />,
	HomeLink: <a href="/" />,
} );
```